dspace-rest/pom.xml

```diff
@@ -69,13 +69,52 @@
                     <groupId>com.fasterxml.jackson.core</groupId>
                     <artifactId>jackson-annotations</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>com.fasterxml.jackson.core</groupId>
+                    <artifactId>jackson-jaxrs-base</artifactId>
```

Comment:
I can find no evidence that these first three excluded artifacts (in com.fasterxml.jackson.core) ever existed.  These can't hurt, but I don't think they are doing anything.